worker.rb

```diff
@@ -27,39 +29,51 @@
       key = URI.decode(record['s3']['object']['key']).gsub('+', ' ')
       log.info "scanning s3://#{bucket}/#{key}..."
       begin
-        resp = s3.get_object(
-          response_target: '/tmp/target',
+        res = s3.head_object(
           bucket: bucket,
           key: key
         )
+        filesize = res.content_length
+        status = 'OK'
+        log.info "s3://#{bucket}/#{key} file size : #{filesize}"
+        if filesize <= LIMIT_SIZE
+          log.info "s3://#{bucket}/#{key} is under the limitation"
+          resp = s3.get_object(
```

Comment:
It would be better to avoid having res and resp variable. Maybe you could rename them head_response and get_response.

Reply:
Done